docs/api-docs/channels/channel-app-requirements.md

```diff
@@ -1,63 +1,68 @@
 
-## General Requirements
+## General requirements
+
+All Partners:
 
-**All Partners**:
-* Uses [Channels API](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api)
-* [Creates a channel ](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels/createchannel) upon app installation using the `app_id`
-* Reads and updates channel status via [Channels endpoints](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api)
-* Follows requirements for specific channel type.
-* Contains an onboarding user flow
+- Uses [Channels API](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api)
+- [Creates a channel](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels/createchannel) upon app installation using the `app_id`
+- Reads and updates channel status via [Channels endpoints](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api)
+- Follows requirements for specific channel type.
+- Contains an onboarding user flow
 
-**Select Partners**:
-* Uses [Big Design](https://developer.bigcommerce.com/big-design/)
+Select Partners:
+
+- Uses [Big Design](https://developer.bigcommerce.com/big-design/)
 
 ## Storefronts
 
-**All Partners**:
-* Must use [Sites & Routes API](https://developer.bigcommerce.com/api-reference/cart-checkout/sites-routes-api) so that links generated within BigCommerce, such as "view storefront" and links sent in transactional emails to shoppers, will use the headless storefront's correct URL.
+All Partners:
+
+- Must use [Sites and Routes API](https://developer.bigcommerce.com/api-reference/cart-checkout/sites-routes-api) so that links generated within BigCommerce, such as "view storefront" and links sent in transactional emails to shoppers, will use the headless storefront's correct URL.
 
-## Marketplaces & Marketing
+## Marketplaces and marketing
 
-**All Partners**:
-* Must use [listings API](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api), if supporting per product listings (i.e. not simply syncing the entire catalog).
+All Partners:
 
-<div class="HubBlock--callout">
-<div class="CalloutBlock--info">
-<div class="HubBlock-content">
+- Must use [listings API](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api), if supporting per product listings (i.e. not simply syncing the entire catalog).
 
 <!-- theme: info -->
 
-### Note
-> * In order to promote app performance and user experience best practices, additional general and channel type requirements will be outlined as needed.
-
-</div>
-</div>
-</div>
+> **Note**

```

Comment:
Were the divs removed on purpose? We want to make sure it renders correctly.

Reply:
Yes, the idea was to avoid mixing HTML and markdown to increase readability. Using <!-- theme: info --> is the recommended method in the Stoplight flavored markdown docs: https://meta.stoplight.io/docs/studio/docs/Documentation/03a-stoplight-flavored-markdown.md

Aside from readability, one issue you might run into is Stoplight changing the div class names between Stoplight versions. While <!-- theme: info --> currently renders into <div class="CalloutBlock--info">	
<div class="HubBlock-content">, there isn't anything to say that will continue to be the case. 

I will change these back to the HTML throughout the docs.